### ISSUE TITLE
options screen

### ISSUE THREAD

**Issue by electronstudio** [electronstudio/MobControlJava#9]
There should be an options screen containing:
* Player name text field
* Player colour1 and colour2 selector
* Sensitivity slider for analogue sticks

One idea I had was when the phone is in portrait mode show the options screen and show the pad only in landscape mode.

**Comment by Nockm** [comment]
Nice idea, I'll have a look at that.

**Comment by Nockm** [comment]
We now have a working interface that emits the following messages:

#### Player name text field

```
{
    "__type__": "set_name",
    "name": "Terry"
}
```

#### Player colour1 and colour2 selector

```
{
    "__type__": "set_colour_1",
    "colour": "green"
}
```

```
{
    "__type__": "set_colour_2",
    "colour": "fuchsia"
}
```

#### Sensitivity slider for analogue sticks

```
{
    "__type__": "set_sensitivity",
    "sensitivity": 0.78
}
```

----

N.B. For the colour selection, HTML colour names are currently used, we can switch over to whichever format by editing this array:

https://github.com/electronstudio/MobControlJava/blob/4f118751bb85a61cd1d0a0ba0fb5fe5eb7741beb/src/main/resources/client/pages/settings.js#L5-L23


Leaving this issue open in case further work is requested.

**Comment by Nockm** [comment]
Sensitivity now affects the padState directly, instead of sending a request. Moving the sensitivity slider from left to right adjusts the extent between the range defined here:

https://github.com/electronstudio/MobControlJava/blob/e3e563f41d7c79670c3204c2f1a43960ae4b29b7/src/main/resources/client/lib/padState.js#L16-L17

The values used in the range should at some point be changed from pixels to screen proportion.

**Comment by electronstudio** [comment]
I guess we should store all these settings in cookies or local storage or whatever its called?

**Comment by Nockm** [comment]
Good idea, next I'll centralise user-config and add stubs for config save/load.
